src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -231,6 +231,7 @@ def load_arguments(self, _):
         c.argument('network_policy')
         c.argument('kube_proxy_config')
         c.argument('auto_upgrade_channel', arg_type=get_enum_type(auto_upgrade_channels))
+        c.argument('kube_proxy_config')
```

Comment:
This is under the scope of `aks create`, please put it under `aks update`.

Reply:
moved